pkg/apis/v1/labels.go

```diff
@@ -119,6 +119,7 @@ var (
 	LabelInstanceAcceleratorManufacturer      = apis.Group + "/instance-accelerator-manufacturer"
 	LabelInstanceAcceleratorCount             = apis.Group + "/instance-accelerator-count"
 	AnnotationEC2NodeClassHash                = apis.Group + "/ec2nodeclass-hash"
+	AnnotationKubeletHash                     = apis.Group + "/kubelet-drift-hash"
```

Comment:
It might make more sense to have a compatibility group here. Similar to upstream https://github.com/kubernetes-sigs/karpenter/blob/c4be45e04079f0d9313150b3ae7b5313132b0e36/pkg/apis/apis.go#L28